### DISCUSSION TITLE
How about adding ModuleInitializer?

### DISCUSSION THREAD

**Discussion by kzu** [manuelroemer/IsExternalInit#7] [Ideas]
In the same vein as Nullable and this project, this attribute is missing in older TFMs and is required to create a module initializer:

```csharp
namespace System.Runtime.CompilerServices
{
    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
    public sealed class ModuleInitializerAttribute : Attribute { }
}
```

C# 9 proposal: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/module-initializers

It's already shipped too :)

**Comment by manuelroemer** [comment]
I will keep this one mind and potentially do it at some point (assuming that nobody else does it in between).
I must admit though that I don't really have a lot of time/motivation right now, so it's probably not something I'd publish in the upcoming days. But anybody is free to fork/adapt this project here, of course. ;)
